libs/SpatialAlignment-Unity/.gitignore

```diff
@@ -0,0 +1,71 @@
+# This .gitignore file should be placed at the root of your Unity project directory
```

Comment:
nit: Pitchfork has the `extras/` directory for language/platform bindings. Should we put Unity-specific projects there rather than in `libs/`?

Reply:
Well this will serve as a "library", so not an "extra".

Reply:
Extras *are* libraries: https://api.csswg.org/bikeshed/?force=1&url=https://raw.githubusercontent.com/vector-of-bool/pitchfork/develop/data/spec.bs#submod.extras. They are just not part of the "core". So I guess it's a matter of deciding if this is core or not.

Reply:
Spoke on IM, leaving as is for now, will revisit in the future.